modules/dcache/src/main/java/diskCacheV111/services/PoolStatisticsV0.java

```diff
@@ -663,7 +610,7 @@ private void updateHtmlTop() throws IOException {
                 StringTokenizer st = new StringTokenizer(line);
 
                 st.nextToken();
-                String key =  _yearOfCalendar.get().format(new Date(Long.parseLong(st.nextToken())));
+                String key = new Date(Long.parseLong(st.nextToken())).toInstant().atZone(ZoneId.systemDefault()).format(_yearOfCalendar);
```

Comment:
To format an Instant a time-zone is required. Without a time-zone, the formatter does not know how to convert the instant to human date-time fields, and therefore throws an exception.

to the formatter _yearOfCalendar append 
```java
.withZone(ZoneId.systemDefault())
```